Trigger/Algo/src/Trigger/Event/TrigCaloHit.cxx

```diff
@@ -3,7 +3,15 @@
 ClassImp(trigger::TrigCaloHit)
```

Comment:
```suggestion
ClassImp(trigger::TrigCaloHit);
```
adding a unnecessary semicolon after this macro helps the formatter not get confused.